tests/storages_tests/test_with_server.py

```diff
@@ -147,7 +149,20 @@ def test_store_nan_intermediate_values(storage: BaseStorage) -> None:
     assert np.isnan(got_value)
 
 
-def test_multiprocess(storage: BaseStorage) -> None:
+def test_multithread_create_study(storage: BaseStorage) -> None:
+    with ThreadPoolExecutor(10) as pool:
+        for _ in range(10):
+            pool.submit(
+                optuna.create_study,
+                storage=storage,
+                study_name="test-multithread-create-study",
+                load_if_exists=True,
+            )
+
+    assert len(storage.get_all_studies()) == 1
```

Comment:
The above tests have created some storages, so we need to refresh DB before this test case or just remove the last assertion. I prefer the latter.

Reply:
Hmmm, instead of removing the last assertion. How about `assert len(storage.get_all_studies()) >= 1`?

Reply:
I'm sorry that my suggested test case is broken.

I agree with Imamura-san's comment. It may cause confusion to readers because the intention of the last assertion is not clear.

Reply:
The number of studies is already more than 1 before starting this test case, so that assertion is a bit strange. Can we just remove the assertion?

Reply:
Ok, I see.